In-house patch to remove unnecessary dependencies from Neutron's
requirements files. The specific reasons are as follows:

amqplib		No longer applicable
		(upstream commit 39ffcb0dd75d9dc5d8a249f2974a67f0def5204b)

argparse	Not applicable to Solaris (Brocade specific)

novaclient	No longer applicable

pyudev		Not applicable to Solaris (Linux bridging specific)

--- neutron-2013.2.3/neutron.egg-info/requires.txt.orig	2014-04-03 11:49:53.000000000 -0700
+++ neutron-2013.2.3/neutron.egg-info/requires.txt	2014-05-24 23:12:26.486505631 -0700
@@ -1,9 +1,7 @@
-pyudev
 pbr>=0.5.21,<1.0
 Paste
 PasteDeploy>=1.5.0
 Routes>=1.12.3
-amqplib>=0.6.1
 anyjson>=0.3.3
 Babel>=1.3
 eventlet>=0.13.0
@@ -23,4 +21,3 @@
 six>=1.4.1
 stevedore>=0.10
 oslo.config>=1.2.0
-python-novaclient>=2.15.0
\ No newline at end of file

--- neutron-2013.2.3/requirements.txt.orig	2014-04-03 11:49:01.000000000 -0700
+++ neutron-2013.2.3/requirements.txt	2014-05-24 23:12:57.116696613 -0700
@@ -3,9 +3,7 @@
 Paste
 PasteDeploy>=1.5.0
 Routes>=1.12.3
-amqplib>=0.6.1
 anyjson>=0.3.3
-argparse
 Babel>=1.3
 eventlet>=0.13.0
 greenlet>=0.3.2
@@ -25,4 +23,3 @@
 stevedore>=0.10
 oslo.config>=1.2.0
 
-python-novaclient>=2.15.0
